ggshield/core/text_utils.py

```diff
@@ -32,8 +32,11 @@
     "line_count_secret": {"fg": "yellow"},
     "progress": {"fg": "bright_yellow", "bold": False},
     "warning": {"fg": "yellow"},
+    "incident_value": {"fg": "bright_yellow", "bold": True},
 }
 
+POLICY_TO_DISPLAY_NAME = {"secrets detection": "Secret detected"}
```

Comment:
Instead of exposing a dict, I think it's nicer to provide a function to return the display name, so that callers do not need to know the details. Something like this (see `translate_validity` for an example):

```python
_DISPLAY_NAME_FOR_POLICY = {
    # ...
}

get_display_name_for_policy(policy: str) -> str:
    return _DISPLAY_NAME_FOR_POLICY.get(policy.lower(), policy)
```

I am also a fan of using `<dst>_for_<src>` instead of `<src>_to_<dst>`, because of https://www.joelonsoftware.com/2005/05/11/making-wrong-code-look-wrong/ (warning, long read!)